### PR TITLE
#811 progress indicator for encoding objects

### PR DESCRIPTION
**Pull request by bigld**

### CODE REVIEW

File: src/zabapgit_git.prog.abap [hunk 1]
```diff
@@ -1043,7 +1043,18 @@ CLASS lcl_git_pack IMPLEMENTATION.
...
+
     LOOP AT it_objects ASSIGNING <ls_object>.
+
+      lv_objects_processed = sy-tabix.
+
+      cl_progress_indicator=>progress_indicate(
+        EXPORTING
+          i_text               = |encoding objects &1% ( &2 of &3 )|
+          i_processed          = lv_objects_processed
+          i_total              = lv_objects_total ).
```
Comment (larshp) [inline src/zabapgit_git.prog.abap]: hi, was thinking perhaps to add code that only updates the progress indicator each second, as this loop is quite fast and showing the progress indicator is expensive.

Comment (bigld) [inline src/zabapgit_git.prog.abap]: FM SAPGUI_PROGRESS_INDICATOR is called only every 10 seconds - this should be ok...

Comment (larshp) [inline src/zabapgit_git.prog.abap]: oh, yeah, sorry

I'm not sure if CL_PROGRESS_INDICATOR exists in 702, but lets try merging it

Comment (bigld) [inline src/zabapgit_git.prog.abap]: Just checked...
image

**Pull request merged**